src/components/serverPaginatedTable/serverPaginatedTable.js

```diff
@@ -188,8 +245,11 @@ class ServerPaginatedTableView extends React.Component {
 
   render() {
     const {
-      data, count, isLoading, rowsPerPage, sortOrder, className,
+      data, count, isLoading, sortOrder, className,
     } = this.state;
+    const stateRowsPerPage = this.state.rowsPerPage;
+    const rowsPerPage = stateRowsPerPage;
```

Comment:
const rowsPerPage = this.state.rowsPerPage;
this will do the trick?

Reply:
This is part of the old code that I did not update completely, and I will update it now.